jmxfetch.py

```diff
@@ -438,6 +438,8 @@ def _is_jmx_check(check_config, check_name, checks_list):
 
             if custom_jar_paths:
                 if isinstance(custom_jar_paths, basestring):
+                    log.warn('Using a string when having only one custom jar will be deprecated in future versions' +
+                        ' of the agent. Only the list syntax will be supported. %s' % LINK_TO_DOC)
```

Comment:
could you use this syntax: `log.warn("foo bar %s", LINK_TO_DOC)` instead? (internally, it only formats the string if the log entry is actually logged, we should try to prefer this syntax as much as possible)